lib/expand/join.go

```diff
@@ -666,19 +665,8 @@ func (p *Peer) checkAndSetServerProfile(app ops.Application) error {
 		"specified node role %q is not defined in the application manifest", p.Role))
 }
 
-// runLocalChecks makes sure node satisfies system requirements
-func (p *Peer) runLocalChecks(cluster ops.Site, installOperation ops.SiteOperation) error {
```

Comment:
Removing local checks as it's difficult to come up with a heuristic when it's safe to run them since expand operation state is not updated beyond `OperationStateExpandPrechecks` and they have become redundant after Roman has introduced cluster<->joining node checks phase.

Reply:
Hmm, they were actually here for a purpose. So when the user launches gravity join (in any case, expand or wizard/cli install) and there are any local problems, it exits immediately and prints failures in the console. This was a UX improvement requested by Ev.

Reply:
Ok, I'll try to come up with a heuristic to only run them at early operation stage.